tokio/src/runtime/scheduler/inject/shared.rs

```diff
@@ -95,7 +95,7 @@ impl<T: 'static> Shared<T> {
     ///
     /// Must be called with the same `Synced` instance returned by `Inject::new`
     pub(crate) unsafe fn pop(&self, synced: &mut Synced) -> Option<task::Notified<T>> {
-        self.pop_n(synced, 1).next()
+        unsafe { self.pop_n(synced, 1).next() }
```

Comment:
This function is very short, and it’s clear enough for developers to reason about the safety requirements, so we don’t need additional "Safety" comments here”